### ISSUE TITLE
TypeError: Cannot read property 'createRelease' of undefined

### ISSUE THREAD

**Issue by MoOx** [coderaiser/node-grizzly#1]
It seems latest release broke something.

```
.../node_modules/grizzly/lib/release.js:45
            github.releases.createRelease({
                           ^

TypeError: Cannot read property 'createRelease' of undefined
    at release (.../node_modules/grizzly/lib/release.js:45:28)
    at .../node_modules/grizzly/lib/release.js:19:13
    at ExecProto.exec (.../node_modules/execon/lib/exec.js:24:40)
    at Function.ExecProto.exec.if (.../node_modules/execon/lib/exec.js:68:17)
    at module.exports (.../node_modules/grizzly/lib/release.js:18:16)
    at Object.<anonymous> (.../node_modules/github-release-from-changelog/github-release-from-changelog.js:110:1)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
```

**Comment by coderaiser** [comment]
Thank you, working on it. Do you have any ideas how it could be fixed?

**Comment by coderaiser** [comment]
Fixed in [v1.0.18](https://github.com/coderaiser/node-grizzly/releases/tag/v1.0.18)  with https://github.com/coderaiser/node-grizzly/commit/f4f4e89482c3564d0057fb1c0adb93c4a18f0789 https://github.com/coderaiser/node-grizzly/commit/b0ec7ced94fb71e45fb0b1db7c876a6a3c5d0db2

**Comment by MoOx** [comment]
Thanks for v2 :)
